packages/react-query/src/internals/getClientArgs.ts

```diff
@@ -9,7 +9,12 @@ export function getClientArgs<TOptions>(
   pageParam?: any,
 ) {
   const path = queryKey[0];
-  const input = queryKey[1]?.input;
-  if (pageParam) (input as any).cursor = pageParam;
```

Comment:
cc @juliusmarminge this caused a bug report a year later 🐛  😅 

let's never mutate any non-local variable

Reply:
took me an hour to find lol, even went on a detour testing if react-query did what it should